src/dawn/engine_dawn.cc

```diff
@@ -811,6 +797,7 @@ Result DawnPipelineHelper::CreateRenderPipelineDescriptor(
 
   renderPipelineDescriptor.primitiveTopology =
       ::dawn::PrimitiveTopology::TriangleList;
+  // TODO(sarahM0): Figure out what sampleCount is, it's been used a few times
```

Comment:
That's for multi-sample rendering:  multiple invocations of the fragment shader (e.g. 4 or 8 or more) are processed with slightly different positions within the final (square) pixel, and the colour results can be blended together. This is used for multi-sample anti-aliasing (sometime MSAA).
See https://www.khronos.org/registry/vulkan/specs/1.1/html/vkspec.html#primsrast-multisampling 
especially the multi-sample locations within a square pixel, mapped out in Table 23 in that edition of the Vulkan spec.